app/components/info-cards/vertical-card.tsx

```
@@ -0,0 +1,72 @@
+  title,
+  subtitle,
+  href,
+  onClick = () => { }
+}: CardProps) {
+  const router = useRouter();
+  return (
+    <div className={`vertical-card song-card rounded-lg bg-[--md-sys-color-outline-variant] flex flex-col items-center justify-center overflow-hidden w-fit gap-3 pb-3`} onClick={
+      () => {
+        router.push(href);
+      }
+    }>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)